src/Illuminate/Foundation/Testing/Concerns/MakesHttpRequests.php

```diff
@@ -274,8 +274,8 @@ protected function transformHeadersToServerVars(array $headers)
         foreach ($headers as $name => $value) {
             $name = strtr(strtoupper($name), '-', '_');
 
-            if (! Str::startsWith($name, $prefix) && $name != 'CONTENT_TYPE') {
-                $name = $prefix.$name;
+            if (!Str::startsWith($name, $prefix) && $name != 'CONTENT_TYPE') {
+                $name = $prefix . $name;
```

Comment:
yeah, i know — it's from the plugin I use. I thought StyleCI could fix this. I'll resubmit this later.  

Thanks